### PR TITLE
fixed spelling errors

### PR DESCRIPTION
**Pull request by rettinghaus**

### CODE REVIEW

File: pae-examples.xhtml [hunk 1]
```diff
@@ -137,13 +137,13 @@ version-footer: true
                 <div class="highlight"><pre><code id="code_pae4"></code></pre></div>
             </p>
 
-            <h4>Grace notes (accaciature)</h4>
+            <h4>Grace notes (accacciature)</h4>
```
Comment (craigsapp) [inline pae-examples.xhtml]: Are you sure it is spelled right now?  This website says it is "acciaccatura" (with the "i" in a different place):
https://en.wiktionary.org/wiki/acciaccatura

"acciaccature" is the Italian plural, but in an English context, using "acciaccaturas" would probably be preferrable.

Comment (rettinghaus) [inline pae-examples.xhtml]: thanks for checking through! I didn't want to change to english on my own, but you're probably right.

File: pae-examples.xhtml [hunk 2]
```diff
@@ -137,13 +137,13 @@ version-footer: true
...
             <div id="svg_pae5" style="overflow:auto;"/>
             <p>
                 <div class="highlight"><pre><code id="code_pae5"></code></pre></div>
             </p>
 
-            <h4>Grace notes (appogiature)</h4>
+            <h4>Grace notes (appoggiature)</h4>
```
Comment (craigsapp) [inline pae-examples.xhtml]: Similar to acciaccature/acciaccaturas, a more preferred plural in English is appoggiaturas:
     https://en.wiktionary.org/wiki/appoggiatura

**Comment by craigsapp** [comment]
English speakers can't handle plurals other than "s" :-), other than continually debating if it should be "octopusses" or "octopi":
     https://en.wiktionary.org/wiki/octopus

**Pull request merged**